src/App.tsx

```diff
@@ -1,204 +1,65 @@
-            <button type="button" className="todo__remove" data-cy="TodoDelete">
-              ×
-            </button>
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
 
-            {/* overlay will cover the todo while it is being deleted or updated */}
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+import { UserWarning } from './UserWarning';
 
-          {/* This todo is an active todo */}
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
+import { Header, TodoList, Footer, ErrorMessage } from './components';
+import { FilterBy } from './types/FilterBy';
+import { filterTodos } from './utils/FilterTodos';
 
-            <span data-cy="TodoTitle" className="todo__title">
-              Not Completed Todo
-            </span>
-            <button type="button" className="todo__remove" data-cy="TodoDelete">
-              ×
-            </button>
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
 
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+  const [errorMessage, setErrorMessage] = useState<Errors | null>(null);
+  const [selectedFilter, setSelectedFilter] = useState<FilterBy>(FilterBy.ALL);
 
-          {/* This todo is being edited */}
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
+  const filteredTodos = filterTodos(todos, selectedFilter);
 
-            {/* This form is shown instead of the title and remove button */}
-            <form>
-              <input
-                data-cy="TodoTitleField"
-                type="text"
-                className="todo__title-field"
-                placeholder="Empty todo will be deleted"
-                value="Todo is being edited now"
-              />
-            </form>
+  const handleCloseErrorButton = () => {
+    setErrorMessage(null);
+  };
 
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => setErrorMessage(Errors.LOAD_ERROR));
```

Comment:
You should remove an error after 3 seconds.
You can create a handler to set and remove error so you can reuse it in 3 seconds
```typescript
const handleError = (error: Errors) => {
  setErrorMessage(error);

  setTimeout(handleCloseErrorButton, 3000);
};
```